dozer-orchestrator/src/pipeline/sinks.rs

```
@@ -318,15 +293,21 @@ pub struct CacheSink {
     // Number of records in the cache that's currently served, if that's different from the one being written to.
     current_alias_count: Option<usize>,
     api_endpoint: ApiEndpoint,
-    pb: ProgressBar,
     notifier: Option<PipelineEventSenders>,
 }
 
 impl Sink for CacheSink {
     fn commit(&mut self) -> Result<(), ExecutionError> {
         let endpoint_name = self.api_endpoint.name.clone();
         // Update Counter on commit
-        self.pb.set_position(self.counter as u64);
+        if let Some(notifier) = &self.notifier {
+            let status_update = StatusUpdate {
+                source: endpoint_name.clone(),
+                r#type: "sink".to_string(),
+                count: self.counter as i64,
+            };
+            try_send(&notifier.2, status_update)?;
```

Should we ignore this error as in `ConnectorSource`?